snykTask/src/__tests__/test-task-args.ts

```diff
@@ -44,6 +44,23 @@ test('ensure no problems if both targetFile and docker-file-path are both not se
   expect(fileArg).toBe('');
 });
 
+test('ensure no problems if testType set to code and both targetFile and dockerImageName are both set', () => {
+  const args = defaultTaskArgs();
+  args.testType = TestType.CODE;
+  args.dockerImageName = 'some-docker-image';
+  args.targetFile = 'some-target-file';
+  args.dockerfilePath = null as any;
+
+  const fileArg = args.getFileParameter();
+  console.log(`fileArg: ${fileArg}`);
+  if (fileArg == null) {
+    console.log('fileArg is null');
+  }
```

Comment:
Could it be that these console.logs were meant to be used only when debugging tests? These logs might not be necessary for the final version, as the expect assertions will already indicate if something is not working as intended. Maybe we could add another assertion:

```
expect(fileArg).not.toBeNull();
```

Reply:
added